RELEASING.md

```diff
@@ -1,8 +1,13 @@
-1. Make certain your branch is in sync with head
+1. Create a new virtual environment following the instructions in `CONTRIBUTING.md`. 
+   Make sure to also install all dependencies for the documentation including `altair_saver`
+   and uninstall `vl-convert-python` (the uninstall step is not needed for normal contributions
+   to the repo, see `CONTRIBUTING.md` for details).
```

Comment:
```suggestion
   and uninstall `vl-convert-python` (this is not needed for normal contributions to the repo, see `CONTRIBUTING.md` for details).
```
I think as such is enough, otherwise we have to explain that the installation of altair_saver is not needed too.

Reply:
To release a new version of the docs, you actually need to install altair_saver, right?

Reply:
Yes, to get all the thumbnails right. 
What I meant is that we should include a line in the text that for normal contributions installing altair_saver and uninstalling vl-convert-python is not necessary. Maybe better to include this line just above the enumeration of actions?

Reply:
Got it, committed your suggestion which looks good to me as part of the first item.